Makefile

```diff
@@ -172,16 +172,30 @@ generate-documentation:
 LIVENESS_PROBE_INITIAL_DELAY_SECONDS ?= 10
 .PHONY: minikube-install
 minikube-install: gadget-default-container kubectl-gadget
+	@echo "Image on the host:"
+	docker image list --format "table {{.ID}}\t{{.Repository}}:{{.Tag}}\t{{.Size}}" |grep $(CONTAINER_REPO):$(IMAGE_TAG)
+	@echo
 	# Unfortunately, minikube-cache and minikube-image have bugs in older
 	# versions. And new versions of minikube don't support all eBPF
-	# features. So we have to keep "docker-save|docker-load" for now.
-	docker save $(CONTAINER_REPO):$(IMAGE_TAG) $(PV) | (eval $(shell $(MINIKUBE) -p minikube docker-env | grep =) && docker load)
+	# features. So we have to keep "docker-save|docker-load" when
+	# available.
+	if $(MINIKUBE) docker-env &>/dev/null ; then \
```

Comment:
I think there is a problem here.

```suggestion
	if $(MINIKUBE) docker-env 2>/dev/null ; then \
```

Current behaviour:
```bash
$ make minikube-install
if minikube docker-env &>/dev/null ; then \
        echo "IF"; \
else \
                echo "ELSE"; \
fi
IF
$
❌  Exiting due to MK_USAGE: The docker-env command is only compatible with the "docker" runtime, but this cluster was configured to use the "containerd" runtime.
```
Expected behaviour:
```bash
$ make minikube-install
if minikube docker-env 2>/dev/null ; then \
        echo "IF"; \
else \
                echo "ELSE"; \
fi
ELSE
```

Reply:
Thanks! This seems to be due to a different shell. It should work if you use `make SHELL=bash`.

I should use `> /dev/null 2>&1` instead of `&>/dev/null`.